### PR TITLE
Re-apply table plugin on table of contents toggle

### PR DESCRIPTION
**Pull request by jbarnicle**
Re-apply table plugin to facilitate resizing when TOC is toggled.

### CODE REVIEW

File: regulations/static/regulations/js/source/views/main/main-view.js [hunk 1]
```diff
@@ -27,8 +27,11 @@ var MainView = Backbone.View.extend({
 
     initialize: function() {
 
+
+        this.dataTable = null;
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/main-view.js]: Would it make sense for this to be an array, instead? I can imagine we'll want to have multiple tables in a single section.

File: regulations/static/regulations/js/source/views/main/main-view.js [hunk 2]
```diff
@@ -27,8 +27,11 @@ var MainView = Backbone.View.extend({
...
         this.render = _.bind(this.render, this);
         this.externalEvents = MainEvents;
+        this.drawerEvents = DrawerEvents;
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/main-view.js]: Doesn't look like this is ever used?

File: regulations/static/regulations/js/source/views/main/main-view.js [hunk 3]
```diff
@@ -37,6 +40,7 @@ var MainView = Backbone.View.extend({
             this.externalEvents.on('breakaway:open', this.breakawayOpen, this);
             this.externalEvents.on('section:error', this.displayError, this);
         }
+        this.externalEvents.on('section:resize', this.redrawTables, this);
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/main-view.js]: If this were `this. applyTablePlugin`, we wouldn't need the `redrawTables` function, hey?

File: regulations/templates/regulations/layers/table.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
     Used to turn markdown tables into HTML tables
 {% endcomment %}
 <div class="table-wrap">
-    <table>
+    <table width="100%">
```
Comment (cmc333333) [inline regulations/templates/regulations/layers/table.html]: Are you sure this is needed? It looks like you've added equivalent CSS in a different PR? If it's needed, please comment why using 
`{% comment %}`

File: regulations/static/regulations/js/source/views/main/main-view.js (hunk 2 above)
Comment (jbarnicle) [inline regulations/static/regulations/js/source/views/main/main-view.js]: You are correct - I will remove.

File: regulations/static/regulations/js/source/views/main/main-view.js (hunk 3 above)
Comment (jbarnicle) [inline regulations/static/regulations/js/source/views/main/main-view.js]: Indeed.  Will refactor.

File: regulations/templates/regulations/layers/table.html (hunk 4 above)
Comment (jbarnicle) [inline regulations/templates/regulations/layers/table.html]: Yea - this (maddeningly) is needed.  If not present, the plugin does not compute and actual width for the scroll container - which is needed to facilitate scrolling via the header area.  This is mentioned in several threads in the Datatables forum.   When you say comment - do you want meek to comment in table.html?

Comment (cmc333333) [inline regulations/templates/regulations/layers/table.html]: Yes, commenting on this line in the template would help explain to future devs why it's here

File: regulations/static/regulations/js/source/views/main/main-view.js (hunk 1 above)
Comment (jbarnicle) [inline regulations/static/regulations/js/source/views/main/main-view.js]: This handles multiple instances as well - the plugin maintains a collection.  I'll pluralize the variable name to make that clearer.

**Comment by cmc333333** [comment]
Cherry-picking https://github.com/cmc333333/regulations-site/commit/6d92c65db8c32dbf43c0880f287f9933d313b5cd will fix travis

Would it be easy to write a test for the diff hash?

**Comment by jbarnicle** [comment]
@cmc333333  - I'll look into writing a test for diff.  I'm not entirely sure what you mean by cherry picking here - I don't have a ton of background around doing merges beyond the full-on merge.  Can you elaborate?

**Comment by cmc333333** [comment]
To cherry-pick is to apply a particular commit (or set of commits) to your branch. In your case, it'd look something like:
```sh
git remote add cm https://github.com/cmc333333/regulations-site.git
git fetch
git cherry-pick 6d92c65db8c32dbf43c0880f287f9933d313b5cd
git push origin master
```

**Comment by cmc333333** [comment]
Thank you kindly

**Pull request merged**